nextflow.config

```diff
@@ -175,6 +175,14 @@ profiles {
     }
     test      { includeConfig 'conf/test.config'      }
 
+    local {
+        params {
+            // Any modern laptop / desktop should have at least...
+            max_memory = "12GB"
```

Comment:
Is it definitely possible to run the pipeline with 12Gb? Have you tested this?

Reply:
I was able to run the pipeline in my laptop with these settings.

Reply:
That's great!